From: miodroid contributors
Date: 2026-10-03
Subject: [PATCH 0001] rebrand: rename waydroid to miodroid throughout

Rename the program name from "waydroid" to "miodroid" so the fork can
coexist with upstream Waydroid. Renames:
  - CLI binary:       waydroid  -> miodroid
  - Config INI key:   [waydroid] -> [miodroid]
  - systemd service:  waydroid-container -> miodroid-container
  - D-Bus name:       id.waydro.Container -> id.miodro.Container
  - Help / version strings updated

--- a/tools/config/__init__.py
+++ b/tools/config/__init__.py
@@ -13,7 +13,7 @@ from tools.config.save import save
 #
 # Exported variables (internal configuration)
 #
-version = "1.6.3"
+version = "1.6.3+miodroid1"
 tools_src = os.path.normpath(os.path.realpath(__file__) + "/../../..")
 
 # Keys saved in the config file (mostly what we ask in 'waydroid init')
--- a/tools/config/load.py
+++ b/tools/config/load.py
@@ -10,9 +10,9 @@ import tools.config
 def load(args):
     cfg = configparser.ConfigParser()
     if os.path.isfile(args.config):
         cfg.read(args.config)
 
-    if "waydroid" not in cfg:
-        cfg["waydroid"] = {}
+    if "miodroid" not in cfg:
+        cfg["miodroid"] = {}
 
-    for key in tools.config.defaults:
-        if key in tools.config.config_keys and key not in cfg["waydroid"]:
-            cfg["waydroid"][key] = str(tools.config.defaults[key])
+    for key in tools.config.defaults:
+        if key in tools.config.config_keys and key not in cfg["miodroid"]:
+            cfg["miodroid"][key] = str(tools.config.defaults[key])
 
-        if key not in tools.config.config_keys and key in cfg["waydroid"]:
+        if key not in tools.config.config_keys and key in cfg["miodroid"]:
             logging.debug("Ignored unconfigurable and possibly outdated"
-                          " default value from config: {}".format(cfg['waydroid'][key]))
-            del cfg["waydroid"][key]
+                          " default value from config: {}".format(cfg['miodroid'][key]))
+            del cfg["miodroid"][key]
 
     if "properties" not in cfg:
         cfg["properties"] = {}
--- a/tools/config/save.py
+++ b/tools/config/save.py
@@ -1,20 +1,20 @@
 # Copyright 2021 Oliver Smith
 # SPDX-License-Identifier: GPL-3.0-or-later
 import configparser
 import os
 import tools.config
 
 def save(args, cfg):
     # Save config
-    if "waydroid" not in cfg:
-        cfg["waydroid"] = {}
+    if "miodroid" not in cfg:
+        cfg["miodroid"] = {}
     for key in tools.config.config_keys:
-        cfg["waydroid"][key] = str(getattr(args, key, tools.config.defaults.get(key, "")))
+        cfg["miodroid"][key] = str(getattr(args, key, tools.config.defaults.get(key, "")))
--- a/tools/helpers/arguments.py
+++ b/tools/helpers/arguments.py
@@ -1,7 +1,7 @@
 def arguments():
-    parser = argparse.ArgumentParser(prog="waydroid")
+    parser = argparse.ArgumentParser(prog="miodroid")
--- a/dbus/id.waydro.Container.conf
+++ b/dbus/id.miodro.Container.conf
@@ -1,11 +1,11 @@
 <!DOCTYPE busconfig PUBLIC
  "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
  "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
 <busconfig>
     <policy user="root">
-        <allow own="id.waydro.Container"/>
+        <allow own="id.miodro.Container"/>
     </policy>
     <policy context="default">
-        <allow send_destination="id.waydro.Container"/>
-        <allow receive_sender="id.waydro.Container"/>
+        <allow send_destination="id.miodro.Container"/>
+        <allow receive_sender="id.miodro.Container"/>
     </policy>
 </busconfig>
--- a/dbus/id.waydro.Container.service
+++ b/dbus/id.miodro.Container.service
@@ -1,5 +1,5 @@
 [D-BUS Service]
-Name=id.waydro.Container
-Exec=/usr/bin/waydroid -w container start
+Name=id.miodro.Container
+Exec=/usr/bin/miodroid -w container start
 User=root
-SystemdService=waydroid-container.service
+SystemdService=miodroid-container.service
--- a/systemd/waydroid-container.service
+++ b/systemd/miodroid-container.service
@@ -1,9 +1,9 @@
 [Unit]
-Description=Waydroid Container
+Description=Miodroid Container
 
 [Service]
 UMask=0022
-BusName=id.waydro.Container
-ExecStart=/usr/bin/waydroid container start
+BusName=id.miodro.Container
+ExecStart=/usr/bin/miodroid container start
 
 [Install]
 WantedBy=multi-user.target
